test/integration/authenticatable_test.rb

```diff
@@ -3,6 +3,12 @@
 require 'test_helper'
 
 class AuthenticationSanityTest < Devise::IntegrationTest
+  test 'sign in should not run model validations' do
+    sign_in_as_user
+
+    refute $validations_performed
```

Comment:
When I said a global I was thinking more in a class variable inside the model

Reply:
I get it now 😅 
I've changed to use a class variable